drop-storage/Cargo.toml

```
@@ -0,0 +1,24 @@
+[package]
+name = "drop-storage"
+version = "1.0.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+thiserror = "*"
+sqlx = { version = "0.6", features = [ "runtime-tokio-rustls", "sqlite", "time", "chrono" ] }
+slog = { workspace = true }
+tokio = { version = "1", features = ["full"] }
```

```suggestion
tokio = { workspace = true, features = ["full"] }
```
Are you sure that you need all the features?

---

this PR was ripped out of a stable branch, these were used in unit tests, however as this PR doesn't contain them, these are unused, nice catch